serializer/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if(${use_installed_dependencies})
         set(CMAKE_INSTALL_LIBDIR "lib")
     endif()
 
-    install(TARGETS serializer EXPORT azure_iot_sdksTargets
```

Comment:
It might be much simpler to have a single `install()` directive in the main CMake file that lists all the targets instead of having everything repeated over and over in each subdirectory:
```
install(
    TARGETS serializer iothub_test iothub_client ...
    EXPORT azure_iot_sdkTargets
    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
    ...
)
```